datafusion/core/src/physical_plan/file_format/parquet/row_groups.rs

```diff
@@ -627,6 +638,56 @@ mod tests {
         );
 
         // TODO: BYTE_ARRAY support read decimal from parquet, after the 20.0.0 arrow-rs release
+        // BYTE_ARRAY: c1 = decimal128(100000, 28, 3), the c1 is decimal(18,2)
+        // the type of parquet is decimal(18,2)
+        let schema =
+            Schema::new(vec![Field::new("c1", DataType::Decimal128(18, 2), false)]);
+        // cast the type of c1 to decimal(28,3)
+        let left = cast(col("c1"), DataType::Decimal128(28, 3));
+        let expr = left.eq(lit(ScalarValue::Decimal128(Some(100000), 28, 3)));
+        let schema_descr = get_test_schema_descr(vec![(
+            "c1",
+            PhysicalType::BYTE_ARRAY,
+            Some(LogicalType::Decimal {
+                scale: 2,
+                precision: 18,
+            }),
+            Some(18),
+            Some(2),
+            Some(16),
+        )]);
+        let pruning_predicate =
+            PruningPredicate::try_new(expr, Arc::new(schema)).unwrap();
+        // we must use the big-endian when encode the i128 to bytes or vec[u8].
+        let rgm1 = get_row_group_meta_data(
+            &schema_descr,
+            vec![ParquetStatistics::byte_array(
+                // 5.00
+                Some(ByteArray::from(500i128.to_be_bytes().to_vec())),
+                // 80.00
+                Some(ByteArray::from(8000i128.to_be_bytes().to_vec())),
+                None,
+                0,
+                false,
+            )],
+        );
+        let rgm2 = get_row_group_meta_data(
```

Comment:
I wonder if it is worth adding a test for row group metadata that has nulls for min and / or max?

Reply:
Do you means we need add `null` value in the decimal column to check min/max?
I think we can refactor the test cases to do that in the follow up pr.

Reply:
> I wonder if it is worth adding a test for row group metadata that has nulls for min and / or max?

do you means the min or the max value is null?

Reply:
we can add the metadata for one row group  like bellow

```
vec![ParquetStatistics::int32(
                None,
                Some(600),
                None,
                0,
                false,
            )],
```
or 
```
vec![ParquetStatistics::int32(
                Some(100),
                None(600),
                None,
                0,
                false,
            )],
```

Reply:
> do you means the min or the max value is null?

Yes this is what I had in mind